### ISSUE TITLE
Add resizing events for browser changes

### ISSUE THREAD

**Issue by curtislisle** [XDATA-Year-3/QCR-mentions-prototype#2]
The network rendering is opened full width of the size of the browser originally, but when the window size enlarges, the rendering doesn't automatically resize its boundaries. Refreshing browser works, but this requires requiry

**Comment by curtislisle** [comment]
This can be low priority.  For demonstration, browser can be set before starting.
